integrator/app/Services/MarvelApi/Characters.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
Is this file a *Controller*, right?

Another point, this file should not be in this folder. We will use the [Domain Design](https://medium.com/@ibrunotome/a-domain-driven-design-ddd-approach-to-the-laravel-framework-18906b3dd473). You can create the *Characters* path (inside *app* path) and put this file inside a `Controller` folder.

Reply:
Would it be better to create the controller using the **CLI**?
```suggestion
php artisan make:controller CharactersController
```

Reply:
Yes for sure!!